res/app/control-panes/dashboard/apps/apps-controller.js

```diff
@@ -16,23 +16,13 @@ module.exports = function ShellCtrl($scope) {
   // TODO: Move this to server side
   // TODO: Android 2.x doesn't support openSetting(), account for that on the UI
 
-  function openSetting(activity) {
-    run('am start -a android.intent.action.MAIN -n com.android.settings/.Settings\\$' +
-    activity)
-  }
```

Comment:
Why did you remove this?

Reply:
Ok I thought this was added by me/us. I will revert it then alongside with `openWiFiSettings` and `openManageApps` that are also part of original code.